data/os/windows/puppet_aio/5.yaml

```diff
@@ -0,0 +1,2 @@
+---
+mcollective::rubypath: "C:/Program Files/Puppet Labs/Puppet/sys/ruby/bin/ruby.exe"
```

Comment:
we can drop this one, we dont support puppet 5 anymore officially

Reply:
Sure, this makes the patch much more simple :wink: